api/python/provisioner/srv/salt/provisioner/api/install.sls

```diff
@@ -15,13 +15,21 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+set_env_vars:
```

Comment:
in current version that state would be executed always regardless of the api_distr type. Could you please move it under if condition since the issue (I believe) would appear only for pip based setups.

Reply:
Done